src/Analysis/GWASV2/Components/Covariates/CovariatesCardsList.jsx

```diff
@@ -2,15 +2,18 @@ import React from 'react';
 import { PropTypes } from 'prop-types';
 import { TeamOutlined, DeleteOutlined } from '@ant-design/icons';
 import { Card } from 'antd';
-import ACTIONS from '../../Shared/StateManagement/Actions';
 
 const { Meta } = Card;
 
-const CovariatesCardsList = ({ covariates, dispatch }) => (
+const CovariatesCardsList = ({
+  covariates,
+  deleteDichotomousCovariate,
+  deleteContinuousCovariate,
```

Comment:
This is a great idea! 

Updated Action.js to contain: 
```DELETE_COVARIATE: 'deleteCovariate',```

reducer.js to contain
```case ACTIONS.DELETE_COVARIATE:
    if ('concept_id' in action.payload) {
      return {
        ...state,
        covariates: state.covariates.filter(
          (covariate) => covariate.concept_id !== action.payload.concept_id,
        ),
      };
    } if ('provided_name' in action.payload) {
      // Used to delete dichotomous covariates:
      return {
        ...state,
        covariates: state.covariates.filter(
          (covariate) => covariate.provided_name !== action.payload.provided_name,
        ),
      };
    }
    throw new Error('Covariate Object missing needed key to delete');
```

And SelectCovariates to use
```        deleteCovariate={(chosenCovariate) => dispatch({
          type: ACTIONS.DELETE_COVARIATE,
          payload: chosenCovariate,
        })}```
        
and referenced deleteCovariate in CovariesCardsList instead of  deleteDichotomousCovariate and
deleteContinuousCovariate.